clrfeatures.cmake

```diff
@@ -25,3 +25,9 @@ endif(NOT DEFINED FEATURE_DBGIPC)
 if(NOT DEFINED FEATURE_INTERPRETER)
   set(FEATURE_INTERPRETER 0)
 endif(NOT DEFINED FEATURE_INTERPRETER)
+
+if(NOT DEFINED FEATURE_NI_BIND_FALLBACK)
```

Comment:
You should move the NOT WIN32 one level up so that the entire block is within it.